doc.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2023 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+Package cloudsqlconn provides several functions for authorizing and encrypting
+connections. These functions can be used with a database driver to
+connect to a Cloud SQL instance.
+
+The instance connection name for a Cloud SQL instance is always in the
+format "project:region:instance".
+
+# Creating a Dialer
+
+To start working with this package, create a Dialer. There are two ways of
+creating a Dialer, which one you use depends on your database driver.
+
+# Postgres
```

Comment:
Should this be `##`?

Reply:
I think godoc only works with header level 1?